.github/dependabot.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+
+updates:
+  - package-ecosystem: pip
+    directories:
+      #- "/analysis_module"
+      #- "/analysis_ui_module"
+      - "/anonymizer_module"
+      - "/collector_module"
+      - "/corrector_module"
+      #- "/opendata_collector_module"
+      - "/opendata_module"
+      - "/reports_module"
+    schedule:
+        interval: "weekly"
+        day: "thursday"
+    groups:
+      python-minor-patch:
+        applies-to: version-updates
+        update-types:
+          - "minor"
+          - "patch"
+      python-major:
+        applies-to: version-updates
+        update-types:
+          - "major"
+    reviewers:
```

Comment:
Lets not do the automatic reviewer part here, since most of the core team is probably not familiar enough with Python to review this. We can just leave it without a reviewer for now and keep an eye on them.